src/foam/nanos/pm/PM.js

```diff
@@ -166,6 +166,12 @@ foam.CLASS({
                 .setIsActive(true)
                 .build();
               alarmDAO.put(alarm);
+
+              AlarmConfig alarmConfig = new AlarmConfig.Builder(x)
```

Comment:
Create.a DAO decorator that does this on AlarmDAO put - adds a config if it does not exist.